rubygem-will_paginate/rubygem-will_paginate.spec

```diff
@@ -17,13 +17,6 @@ Requires:      %{?scl_prefix_ruby}ruby(rubygems)
 Requires:      %{?scl_prefix_ror}rubygem(activerecord)
```

Comment:
These dependencies are odd. The gemspec doesn't list any but they do have gemfiles in the git root for rails versions. Is this required or not?